app/frontend/src/features/auth/jail/Jail.tsx

```diff
@@ -28,6 +26,7 @@ export default function Jail() {
   const isAuthenticated = authState.authenticated;
 
   const [loading, setLoading] = useState(false);
+  // eslint-disable-next-line
```

Comment:
Disabled warnings because CI would have problems with them and we probably want to reuse this component in the future. But since Aapeli said, that I should left TOS and Map in jail I'll remove it.

Reply:
What warnings do you get? Seems like a pretty bog standard `useState` to me so shouldn't complain?

Reply:
Probably "unused" warning